src/connection/database/login_pane.jsx

```diff
@@ -70,7 +66,12 @@ export default class LoginPane extends React.Component {
       <div>
         {header}
         {fieldPane}
-        {passwordPane}
+        <PasswordPane
```

Comment:
Why are you switching from using a `const`? Not a blocker, just curious

Reply:
the const was a conditional. It was adding/removing the input from the DOM based on that condition. This triggered the bug itself. So, now, it will always have the input, it will only use css to hide the input